test/duke/command/UserCommandsTest.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class UserCommandsTest {
+
+    @Test
+    void testGetTodoCommandWord() {
```

Comment:
test cases should be in featureUnderTest_testScenario_expectedBehavior() format even if it is a assertion of a hardcoded enumeration